src/v/cluster/tests/serialization_rt_test.cc

```diff
@@ -15,6 +15,7 @@
 #include "model/compression.h"
```

Comment:
> cluster: Add expected_pid to internal rpc

please address topics in the commit message related to changes to rpc messages like upgrade/compat/etc...